packages/cli/src/commands/database/seed/tables.ts

```diff
@@ -156,13 +156,19 @@ export const seedTables = async (
   // Create tenant application role
   const applicationRole = createTenantApplicationRole();
   await connection.query(insertInto(applicationRole, Roles.table));
+
+  const m2mAppOnlyScopes = new Set<string>([
+    CloudScope.SendSms,
+    CloudScope.SendEmail,
+    CloudScope.CleanupOutdatedLogs,
```

Comment:
do we need use cloud service app for this? can we just implement a cloud api to clean up all stale audit logs?